src/aap_eda/settings/default.py

```diff
@@ -728,7 +728,15 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     "VALIDATE_HTTPS": settings.get("RESOURCE_SERVER__VALIDATE_HTTPS", False),
 }
 RESOURCE_JWT_USER_ID = settings.get("RESOURCE_JWT_USER_ID", None)
-RESOURCE_SERVICE_PATH = settings.get("RESOURCE_SERVICE_PATH", None)
+
+try:
+    service_path_default = RESOURCE_SERVICE_PATH  # noqa
+except NameError:
+    service_path_default = None
```

Comment:
Would it be clearer if this logic is encapsulated in the default value of dynaconf? That would be in line of how we do it for other settings
For example:
```
def _get_dab_resouce_service_path() -> Optional[str]:
    try:
        service_path_default = RESOURCE_SERVICE_PATH  # noqa
    except NameError:
        service_path_default = None
RESOURCE_SERVICE_PATH = settings.get(
    "RESOURCE_SERVICE_PATH", _get_dab_resouce_service_path()
)
```

Reply:
I tried this but it never felt right to me, because the method will be catching a `NameError` for a global variable, and that seems a little fragile. The way we load the settings into the current module namespace inherently makes things awkward. I put up another form of this in https://github.com/ansible/eda-server/pull/1061 that I still think reads slightly better.